service/src/main/resources/changelog/changesets/20230118_assemble_refbased_prepopulation.yaml

```diff
@@ -0,0 +1,1597 @@
+databaseChangeLog:
+  - changeSet:
+      id: "method prepopulation for assemble_refbased"
+      author: kpierre
+      changes:
+        - insert:
+            tableName: method
+            columns:
+              - column:
+                  name: method_id
+                  value: 00000000-0000-0000-0000-000000000005
+              - column:
+                  name: name
+                  value: assemble_refbased
+              - column:
+                  name: description
+                  value: assemble_refbased
+              - column:
+                  name: created
+                  valueDate: "CURRENT_TIMESTAMP()"
+              - column:
+                  name: method_source
+                  value: Github
+              - column:
+                  name: last_run_set_id
+                  value: null
+        - insert:
+            tableName: method_version
+            columns:
+              - column:
+                  name: method_version_id
+                  value: 50000000-0000-0000-0000-000000000005
+              - column:
+                  name: method_id
+                  value: 00000000-0000-0000-0000-000000000005
+              - column:
+                  name: method_version_name
+                  value: '1.0'
+              - column:
+                  name: method_version_description
+                  value: 'Target workflow assemble_refbased'
```

Comment:
It might be good to remove `Target workflow` from here as well. Should we call it `Example  assemble_refbased` or `Sample assemble_refbased`?